tox.ini

```diff
@@ -106,7 +106,8 @@ description = Run integration tests
 deps =
     aiohttp
     asyncstdlib
-    juju
+    # Libjuju needs to track the juju version
+    juju ~= 3.0.0
```

Comment:
```suggestion
    juju > 3.1
```

Reply:
This doesn't work. You need libjuju 3.0.* for juju 3.0.*

Reply:
`<3.1` is also libjuju 3.0.4. [There aren't other options](https://pypi.org/project/juju/#history). It's functionally exactly the same as `~= 3.0.0`, just without syntax which confuses anyone who has ever used `~=` in JS/Ruby/Perl/shell, where `=~ || ~=` is generally a regexp matcher, including in our own PromQL/LogQL.

It's not a requirement and more of a nit because "`~=` means to stay in this semver series" is a bizarre, requirements.txt-only thing